.tekton/tasks/get-pr-number.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
If all you need is the PR #, that seems a bit convoluted. I would expect it is a parameter that you can get from PaC.

Reply:
No pr number is only populated for `pull_request` event type from pac, for `push` they don't have such https://pipelinesascode.com/docs/guide/authoringprs/